src/common/options.cc

```diff
@@ -7047,6 +7047,10 @@ std::vector<Option> get_mds_client_options() {
     .set_default(false)
     .set_description("big_writes is deprecated in libfuse 3.0.0"),
 
+    Option("fuse_max_write", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(131072)
```

Comment:
The default should be `0` which indicates we do not change whatever the FUSE default is. i.e. we do not pass `-o max_write...`.

Reply:
I separate fuse_max_write from fuse_big_writes. 
I originally wanted to use this switch by default, but considering that fuse has set the default features, then we do not turn on, only when the user sets the fuse_max_write value.